### DISCUSSION TITLE
Link to form submission

### DISCUSSION THREAD

**Discussion by mscruse** [statamic/cms#6474] [Q&A] [answered]
I have the need to put a link to a form submission in the email sent after submission. Rather than put all submitted data in the email, it would be great to add some data then a link directly to the submission in CP

eg: https://<domain>/cp/forms/<form slug>/submissions/<submission id>

Has anyone done this or have ideas on how to achieve it?

**Comment by ryanmitchell** [comment, marked as answer]
You should be able to use the variables available in the email.

{{ id }} - the submission id
{{ form:handle }} - the form handle

So something like:

`https://yourdomain.com/cp/forms/{{ form:handle }}/submissions/{{ id }}`

**Reply by mscruse** [reply]
Brilliant, thanks so much - that's perfect.
